### PR TITLE
OSDOCS-2329: Rel Notes for GCP Workload Identity

### PR DESCRIPTION
**Pull request by jeana-redhat**
Rel notes for [OSDOCS-2329](https://issues.redhat.com/browse/OSDOCS-2329)

Preview: https://deploy-preview-42685--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes#ocp-4-10-auth-gcp-workload-identity

### CODE REVIEW

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -807,6 +807,13 @@ Previously, `oc` commands that used the registry configuration, for example `oc
 
 Users also have the option to use the `REGISTRY_AUTH_FILE` environment variable, which serves as an alternative to the existing `--registry-config` CLI flag. The `REGISTRY_AUTH_FILE` environment variable is also compatible with `podman`.
 
+[id="ocp-4-10-auth-gcp-workload-identity"]
+==== Support for Google Cloud Platform Workload Identity
+
+You can now use the Cloud Credential Operator (CCO) utility `ccoctl` to configure the CCO to use the Google Cloud Platform Workload Identity. When the CCO is configured to use GCP Workload Identity, it assigns IAM service accounts that provide short-term, limited-privilege security credentials to components.
```
Comment (akhil-rane) [inline release_notes/ocp-4-10-release-notes.adoc:813]:
```suggestion
You can now use the Cloud Credential Operator (CCO) utility `ccoctl` to configure the CCO to use the Google Cloud Platform Workload Identity. When the CCO is configured to use GCP Workload Identity, the in-cluster components can impersonate IAM service accounts using short-term, limited-privilege credentials.
```

**Review by akhil-rane — COMMENTED** [review]
/lgtm

**Review by opayne1 — COMMENTED** [review]
lgtm!

**Comment by jeana-redhat** [comment]
Force pushed to force rebuild before merging, no change.

**Pull request merged**